utilities/gitopsctl/implementations/download-job/download-urls-multithreaded.go

```diff
@@ -0,0 +1,47 @@
+package downloadjob
+
+import "fmt"
+
+type downloadURLWorkerEntry struct {
+	url  string
+	path string
+}
+
+func worker(id int, jobs <-chan downloadURLWorkerEntry, results chan<- string) {
+	for jobEntry := range jobs {
+		// fmt.Println("worker", id, "started  job", jobEntry)
```

Comment:
Maybe change the two commented out Println's into log statements or get rid of them?